v2/internal/controllers/crd_eventgrid_topic_test.go

```diff
@@ -116,6 +116,9 @@ func Topic_Subscription_CRUD(tc *testcommon.KubePerTestContext, rg *resources.Re
 		},
 	}
 
+	// Don't try to delete directly, this is not a real resource - to delete it you must delete its parent
+	tc.AddAnnotation(&queueService.ObjectMeta, "serviceoperator.azure.com/reconcile-policy", "detach-on-delete")
```

Comment:
Should we either add an explicit test, or just purposefully leave this out of one of these tests, to ensure that the test still passes?

Possibly an explicit test is better?

Create rg, resource, create child that can't be deleted, delete child. Then delete parent and assert that both parent/child end up deleted in Kuberentes (RG is still there). Then delete RG and clean up?